client/components/partySize.jsx

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
 
 const partySize = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20];
+const spaces = "   ";
```

Comment:
What's the purpose of this variable and are you using it anywhere? Couldn't find it being used.